src/styles/themes/default.js

```diff
@@ -18,6 +18,7 @@ export default {
     textReversed: colors.white,
     textMutedReversed: colors.gray3,
     buttonSuccessBG: colors.green,
+    gray2: colors.gray2,
```

Comment:
This is still incorrect. Please create a new `themeColor` of `buttonDisabledBG` - so instead of `gray2:` it should say `buttonDisabledBG:`. Then please change the `themeColor` on line 270 of `styles.js`

Reply:
Okay got it, Now fixed that.